docs/MultisiteExamples.md

```diff
@@ -106,6 +104,8 @@ spec:
             - zone-1a
 EOF
 ```
+You can have similar CRs for other sites of IndexerCluster by changing the `site` parameter according to the zone label.
+
```

Comment:
Alternate text for consideration:
"Create IndexerCluster CR for each required `site` with zone affinity specified as needed"